src/app/containers/MediaPlayer/helpers/logEmbedSourceStatus/index.js

```diff
@@ -0,0 +1,20 @@
+import nodeLogger from '#lib/logger.node';
+import { MEDIA_PLAYER_RESPONSE } from '#lib/logger.const';
+
+const logger = nodeLogger(__filename);
+
+async function logEmbedSourceStatus(embedSource) {
+  const response = await fetch(embedSource, { method: 'HEAD' });
+  const { status } = response;
+  const data = {
+    url: embedSource,
+    status,
+  };
+  if (status >= 300 || status < 200) {
+    logger.warn(MEDIA_PLAYER_RESPONSE, data);
```

Comment:
Should this be an error rather than a warning?

Reply:
At the amigos session with @graemelindsayBBC @pharingee  and @gavinspence we agreed on using warn instead of error.

Reply:
Thanks 👍